aws/resource_aws_eks_node_group.go

```diff
@@ -357,7 +358,7 @@ func resourceAwsEksNodeGroupRead(d *schema.ResourceData, meta interface{}) error
 	}
 
 	if err != nil {
-		return fmt.Errorf("error reading EKS Node Group (%s): %w", d.Id(), err)
+		return diag.Errorf("error reading EKS Node Group (%s)", d.Id(), err)
```

Comment:
Still need to pass `error`, just can't use `%w`.

```suggestion
		return diag.Errorf("error reading EKS Node Group (%s): %s", d.Id(), err)
```